docs/how-to-work-on-coding-challenges.md

```diff
@@ -251,7 +252,7 @@ We track how long it takes for campers to solve challenges and use this informat
 
 Each challenge should teach exactly one concept, and that concept should be apparent from the challenge's name.
 
-We can reinforce previously covered concepts through repetition and variations - for example, introducing h1 elements in one challenge, then h3 elements a few challenges later.
+We can reinforce previously covered concepts through repetition and variations, for example, introducing h1 elements in one challenge, then h3 elements a few challenges later.
```

Comment:
I would split this sentence up. I don't think that putting a comma here is correct grammar. As a general rule, commas should be used sparingly. They aren't replacements for conjunctions.

> We can reinforce previously covered concepts through repetition and variation. For example, we can introduce h1 elements in one challenge and h3 elements a few challenges later.